reactioncommerce/reaction
# [WIP] Remove Hard Reload From PDP After Publishing changes

#**Resolves**
#2246 

**Testing**
1. Login as admin
1. Create a new product.
1. Set variant label to something
1. Click the edit button next to the social icons
1. Open the product settings accordion
1. Set the product title (and the perma-link will be set as well)
1. Hit publish
1. Observe that the PDP does not reload.
